app/src/main/java/com/example/weather/data/network/api/ForecastApi.java

```diff
@@ -0,0 +1,41 @@
+package com.example.weather.data.network.api;
+
+import com.example.weather.data.db.entity.ForecastData;
+
+import io.reactivex.Observable;
+import okhttp3.OkHttpClient;
+import okhttp3.logging.HttpLoggingInterceptor;
+import retrofit2.Retrofit;
+import retrofit2.adapter.rxjava2.RxJava2CallAdapterFactory;
+import retrofit2.converter.gson.GsonConverterFactory;
+import retrofit2.http.GET;
+import retrofit2.http.Query;
+
+public interface ForecastApi {
+    static String DOMAIN= "http://api.openweathermap.org/";
```

Comment:
Better to rename to BASE_URL and extract to separate file named Consts for all app constants.